intel_extension_for_transformers/neural_chat/examples/tts_finetuning/inference_peppa.py

```diff
@@ -0,0 +1,134 @@
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from transformers import SpeechT5Processor, SpeechT5ForTextToSpeech
+from datasets import load_dataset, Audio, Dataset, Features, ClassLabel
+import os
+import torch
+from speechbrain.pretrained import EncoderClassifier
+from dataclasses import dataclass
+from typing import Any, Dict, List, Union
+from transformers import SpeechT5HifiGan
+import soundfile as sf
+from datetime import datetime
+from num2words import num2words
+
+workdir = os.getcwd()
+
+model = torch.load("peppa.pt")
+processor = SpeechT5Processor.from_pretrained("microsoft/speecht5_tts")
+
+spk_model_name = "speechbrain/spkrec-xvect-voxceleb"
+device = "cuda" if torch.cuda.is_available() else "cpu"
+speaker_model = EncoderClassifier.from_hparams(
+    source=spk_model_name,
+    run_opts={"device": device},
+    savedir=os.path.join("/tmp", spk_model_name)
+)
+def create_speaker_embedding(waveform):
+    with torch.no_grad():
+        speaker_embeddings = speaker_model.encode_batch(torch.tensor(waveform))
+        speaker_embeddings = torch.nn.functional.normalize(speaker_embeddings, dim=2)
+        speaker_embeddings = speaker_embeddings.squeeze().cpu().numpy()
+    return speaker_embeddings
+
+audio_dataset = Dataset.from_dict({"audio": [os.path.join(workdir, "audios/4.mp3")]}).cast_column("audio", Audio(sampling_rate=16000))
+sembeddings = create_speaker_embedding(audio_dataset[0]["audio"]['array'])
+speaker_embeddings = torch.tensor(sembeddings).unsqueeze(0)
+vocoder = SpeechT5HifiGan.from_pretrained("microsoft/speecht5_hifigan").cuda()
+
+def correct_abbreviation(text):
```

Comment:
Can we use the neural_chat tts code to do inference?

Reply:
Will cause _pickle.UnpicklingError if I want to refer to that code